.github/workflows/installer-build-check.yml

```diff
@@ -0,0 +1,243 @@
+name: Build and Test
+
+# Job will run nightly at 03:05 EDT / 02:05 CDT
+# Time below is UTC
+on:
+  schedule:
+    - cron: "5 7 * * *"
+  workflow_dispatch:
+  push:
```

Comment:
I'd suggest not making a `gha-scheduled-*` branch pattern for this to trigger by. Instead just have the cron run and the manual run option. The manual run option would let people run it with whatever branches they want on demand, without having to know about the magical `gha-scheduled-` pattern. Plus, as we have more scheduled jobs, we dont want all of them firing just because we made a branch with a pattern.